package.json

```diff
@@ -16,6 +17,7 @@
     "predeploy": "sh scripts/predeploy.sh",
     "restart-dyno": "node bin/restart",
     "start": "node --max_old_space_size=4096 -r dotenv/config src/index.js",
+    "start:prod": "node --max_old_space_size=4096 -r dotenv/config server.dist.js",
```

Comment:
I added this for testing, but might want to clean up all of these `start` scripts (and below). `start-dev` and `start-prod` are just pointers to other env files as an example of how its done.